opentech/static_src/src/javascript/apply/fancybox-global.js

```diff
@@ -18,4 +18,16 @@
         $('.django-select2-checkboxes').select2('close');
     });
 
+    $(document).ready(
+        $('.modal').each((idx, element) => {
+            var modal = $(element);
+            var error = modal.has('.errorlist');
+            if ( error.length ) {
```

Comment:
Can we remove the spaces within the parenthesis to appease the linter?

Reply:
this was merged fixed in #957 and merged